### ISSUE TITLE
How to locate discussion board for CPP 529 on GitHub

### ISSUE THREAD

**Issue by Jigarci3** [DS4PS/cpp-529-master#12]
Hello,
I am hoping someone can walk me through how to get to the discussion boards on GitHub for each of our labs. I am able to locate the cpp-529-master but can't figure out where the the discussion is happening for each lab. I see that there is a folder for labs but still can't locate the discussion.
Sorry if I am missing the instructions somewhere!

Thanks!

**Comment by lecy** [comment]
The discussion boards were closed, making them hard to find. Can you see them now?

**Comment by Jigarci3** [comment]
@lecy Yes, I am able to see Lab 01 and 02 discussions now on the Get Help page. Just to confirm, there is no discussion items posted for lab 03? 
Thank you!

**Comment by lecy** [comment]
Not that I know of. Feel free to start one, just be sure to include @Anthony-Howell-PhD to make sure he gets a notification.
